src/saveload/saveload.h

```diff
@@ -340,6 +340,7 @@ enum SaveLoadVersion : uint16 {
 	SLV_LINKGRAPH_TRAVEL_TIME,              ///< 297  PR#9457 v12.0-RC1  Store travel time in the linkgraph.
 	SLV_DOCK_DOCKINGTILES,                  ///< 298  PR#9578 All tiles around docks may be docking tiles.
 	SLV_REPAIR_OBJECT_DOCKING_TILES,        ///< 299  PR#9594 v12.0  Fixing issue with docking tiles overlapping objects.
+	SLV_BUILD_OBJECT_BY_AREA,               ///< 300  PR#9709 Build objects by area
```

Comment:
Since SLV_TABLE_CHUNKS (#9322) savegame versions are no longer needed when adding/removing things from savegames. They are only needed if an actual conversion is required.
So I think you can remove the SLV_xxx and all savegame_version_from/to thingies.